hw3/task3/hw3.py

```diff
@@ -0,0 +1,33 @@
+# I decided to write a code that generates data filtering object from a list of keyword parameters:
+
+
+class Filter:
+    """
+    Helper filter class. Accepts a list of single-argument
+    functions that return True if object in list conforms to some criteria
+    """
+
+    def __init__(self, functions):
+        self.functions = functions
+
+    def apply(self, data):
+        return [item for item in data if all(i(item) for i in self.functions)]
+
+
+# example of usage:
+# positive_even = Filter(lambda a: a % 2 == 0, lambda a: a > 0, lambda a: isinstance(int, a)))
+# positive_even.apply(range(100)) should return only even numbers from 0 to 99
+
+
+def make_filter(**keywords):
```

Comment:
Could you please add typings for the code?

Reply:
Done